### DISCUSSION TITLE
About commercial license

### DISCUSSION THREAD

**Discussion by xonep** [saadeghi/daisyui#3327] [Q&A] [answered]
I’m planning to create a SaaS web site builder, and I’d like to know if daisy.ui can be used for commercial purposes in this context. Also, regarding the MIT license credit, would it be acceptable to include it in the HTML comment section instead of displaying it on the site itself?

**Comment by saadeghi** [comment, marked as answer]
Yes! you can use daisyUI for personal or commercial projects.  

If you are using daisyUI as a dependency, there's no need to do anything.  
When you install the package (`npm i daisyui`), the license file is there: https://unpkg.com/browse/daisyui/ already and it stays next with the package in /node_modules.

If you are manually copy/pasting a specific part of daisyUI styles (not as a dependency) to somewhere else, you should also put the `LICENSE` text there, or a link to the license file or the original GitHub repo.

Also, using HTML comment is acceptable. No need for a visible link on the page.

Let me know if you have a question.

**Reply by xonep** [reply]
Thanks so much for your comment ! Everything is clear now !
